common/pango.c

```diff
@@ -109,6 +109,24 @@ void get_text_size(cairo_t *cairo, const char *font, int *width, int *height,
 	free(buf);
 }
 
+void get_text_metrics(const char *font, size_t *height, size_t *baseline) {
```

Comment:
Nit: since `height` and `baseline` are not array indices, I don't think it makes sense for them to be `size_t`. Pango just uses `int`.

Edit: hmm, seems like we incorrectly use it elsewhere. Would be nice to clean this up (but it's unrelated to this PR).